src/cmd/initContainer.go

```diff
@@ -349,8 +361,6 @@ func initContainer(cmd *cobra.Command, args []string) error {
 			logrus.Warnf("Received an error from the file system watcher: %v", err)
 		}
 	}
-
-	return nil
```

Comment:
Ditto as with the removed panic.

Reply:
This code is even more weird than the `panic` in `runCommandWithFallbacks`.

Without the `return`, a casual reader, including me, would think that this is a function that's supposed to return an `error` but isn't returning anything.  It's really not obvious why that would be correct.

Ideally, we should handle `SIGTERM` and have the container gracefully terminate.  Otherwise we should either restore the `return` or add a `panic`.

Reply:
In the end, I went for a comment.  See https://github.com/containers/toolbox/pull/1185 for the full explanation.